src/Instance/InstanceDetails/InstanceAdministrativeView/InstanceAdministrativeView.js

```diff
@@ -75,10 +78,16 @@ const InstanceAdministrativeView = ({
 
       <Row>
         <Col xs={3}>
-          <KeyValue
-            label={<FormattedMessage id="ui-inventory.instanceHrid" />}
-            value={checkIfElementIsEmpty(instance.hrid)}
-          />
+          <KeyValue label={<FormattedMessage id="ui-inventory.instanceHrid" />}>
+            {
+              Boolean(instance.hrid) && (
```

Comment:
@NikitaSedyx I'm getting this error when `hrid` is missing.

````
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
````

Would you mind taking a look at it?